### ISSUE TITLE
Tooltips content should be copiable after the recent changes

### ISSUE THREAD

**Issue by dufipl** [ckeditor/ckeditor5#16110]
## 📝 Provide detailed reproduction steps (if any)

1. Run the editor with the latest master or nightly version e.g. `0.0.0-nightly-20240327.0`
2. Hover a toolbar button and try to copy the tooltip content.

### ✔️ Expected result

Tooltip content is copiable after the recent changes: https://github.com/ckeditor/ckeditor5/pull/16103

### ❌ Actual result

The tooltip disappears when clicked:

https://github.com/ckeditor/ckeditor5/assets/32898551/333042fc-a8a9-4b38-a189-e2c6b20f57ab

## 📃 Other details

* Browser: Any
* OS: MacOS
* Related to those changes: https://github.com/ckeditor/ckeditor5/pull/16103

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by Mati365** [comment]
@dufipl After discussion, we decided to disable selection in tooltip content. It'll not affect our accessibility (and VPAT report) and will be much easier to implement at this moment.
